container/broker/kvm-broker.go

```diff
@@ -18,7 +18,7 @@ import (
 	"github.com/juju/juju/network"
 )
 
-var kvmLogger = loggo.GetLogger("juju.provisioner.kvm")
+var kvmLogger = loggo.GetLogger("juju.container.broker.kvm")
```

Comment:
If people look out/script against debug-log output should we make them aware of this?